developer_manual/core/translation.rst

```diff
@@ -34,7 +34,7 @@ Translators will translate:
 * ' or "
 * cloud
 
-By translating these individual strings for sure the case will be lost of local filesystem and cloud. The two white spaces with the or will get lost while translating as well.
+By translating these individual strings for sure the case will be lost of local filesystem and cloud. The two white spaces with the or will get lost while translating as well. For languages that have a different grammatical order it prevents the translators from reordering the sentence components.
```

Comment:
Yes, it should have some method of quoting like your example. And the reorganising of the English of the two sentences is also much more straightforward.
Since you already know the magic of that highlighting I suggest that you commit my change then make our change directly afterwards.